File: src/main/java/com/telegram/folobot/mappers/FoloVarMapper.java
package com.telegram.folobot.mappers;

import com.telegram.folobot.domain.FoloVarEntity;
import com.telegram.folobot.dto.FoloVarDto;
import org.springframework.stereotype.Component;

@Component
public class FoloVarMapper {
    public FoloVarDto mapToFoloVarDto(FoloVarEntity entity){
        return new FoloVarDto()
                .setId(entity.getId())
                .setValue(entity.getValue());
    }

    public FoloVarEntity mapToFoloVarEntity(FoloVarDto dto){
        return new FoloVarEntity()
                .setId(dto.getId())
                .setValue(dto.getValue());
    }
}

File: src/main/java/com/telegram/folobot/mappers/FoloVarMapper.kt
package com.telegram.folobot.mappers

import com.telegram.folobot.domain.FoloVarEntity
import com.telegram.folobot.dto.FoloVarDto
import org.springframework.stereotype.Component

@Component
class FoloVarMapper {
    fun mapToFoloVarDto(entity: FoloVarEntity): FoloVarDto {
        return FoloVarDto(entity.id, entity.value)
    }

    fun mapToFoloVarEntity(dto: FoloVarDto): FoloVarEntity {
        return FoloVarEntity(dto.id, dto.value)
    }
}